### ISSUE TITLE
Microsoft Defender / Antivirus detections removed in new releases 

### ISSUE THREAD

**Issue by AnthoLaMalice** [WithSecureLabs/chainsaw#168]
Hey guys, 

I have observed that the latest version of Chainsaw no longer seems to report Microsoft Defender/AV detection. 

I ran both v2.9.0 and v2.8.0 on the same log set, which I know contains Microsoft Defender detection for CVE-2021-31207. The default raw output was redirected to a file for testing. 

 v2.9.0 vs v2.8.0 : 

![image](https://github.com/WithSecureLabs/chainsaw/assets/78861013/f1181667-091d-4bc9-91c3-4cf2f1a6016a)

As you can see v2.8.0 indeed showed Microsoft Defender detection which is not the case for v2.9.0.

It also seems that with version 2.8.0, if you output your results to a csv or json file, a specific file has been created for AV detection, which is not the case with version 2.9.0.

Is there an explanation for this? 

Thanks for your work!

**Comment by FranticTyping** [comment]
Hey @AnthoLaMalice 

Thanks for flagging this. I'll take a look next week and get back to you after I've figured out what's going on.

**Comment by alexkornitzer** [comment]
Does undoing this https://github.com/WithSecureLabs/chainsaw/commit/9e04039d571e64d8ef828be284bae2f2127a2860 change to the Chainsaw `windows_defender.yml` rule fix the behaviour?

**Comment by AnthoLaMalice** [comment]
It indeed seems like it fixed the issue : 
![image](https://github.com/WithSecureLabs/chainsaw/assets/78861013/d3425cc4-3fe4-4baf-9012-cf8ee9fe4947)

**Comment by alexkornitzer** [comment]
Awesome, okay that should not break it but now we know where to look.

**Comment by reece394** [comment]
I tried to reproduce this using the same rules but only switching the Chainsaw version on Windows between v2.8.1 and v2.9.0 but I was unable to. They produced identical results apart from a few lines changing positions on the csv which is expected. 1116 and 1117 events appeared correctly using EVTX Attack Samples to test. 

I noticed in the screenshots you were using Linux so this may be a platform specific bug?

**Comment by alexkornitzer** [comment]
@AnthoLaMalice are you able to provide the event log so that I can try and replicate this behaviour?

@reece394 thank you for doing some further triage.

**Comment by alexkornitzer** [comment]
I am not able to replicate this using the EVTX samples. Maybe additionally you could list the command you ran?

<img width="887" alt="Screenshot 2024-05-27 at 13 43 12" src="https://github.com/WithSecureLabs/chainsaw/assets/2750747/b3aa5271-17c5-4277-8776-c7d2e22dface">

**Comment by alexkornitzer** [comment]
Yep, not able to replicate on my machine using the example EVTX files.
<img width="919" alt="Screenshot 2024-05-27 at 14 21 48" src="https://github.com/WithSecureLabs/chainsaw/assets/2750747/8d4d92d0-c83f-4e3e-8d77-05d4cc462f5a">

**Comment by Fuzzdkk** [comment]
Did a quick re-check on this since it's been quiet for a while.

Tested on chainsaw 2.15.0 (current main, commit d8924d6) on Windows, using the `WinDefender_Events_1117_1116_AtomicRedTeam.evtx` sample from EVTX-ATTACK-SAMPLES, which is what reece394 used back in 2024.

```
chainsaw hunt WinDefender.evtx -r rules/evtx
[+] 11 Detections found on 11 documents
```

All 11 events (mix of 1116 and 1117) fire under the Antivirus group. CSV output also creates `antivirus.csv` as expected, and JSON includes the same hits, so the missing AV file behaviour mentioned in the original report doesn't seem to be present either.

@AnthoLaMalice if this is still happening for you on a recent build, would you mind dropping the EVTX (or a redacted snippet) so it can be looked at? Otherwise this looks closeable given the silence and three negative repros now (reece394 on Windows, Alex on his setup, and me on Windows 2.15.0).

**Comment by alexkornitzer** [comment]
This is very stale and not really in bug status anymore so i'll close it and if it can be reproduced it can be opened again.
